scripts/build-full-local.sh

```diff
@@ -0,0 +1,24 @@
+# Add your Joplin PR API and Media Links here...
+# - NOTE: make sure CMS_API has `/api/graphiql` endpoints
+# - AND... CMS_MEDIA has `/media` endpoints
+
+# Examples
+# - export CMS_API='https://joplin-pr-3244-guide-icon-tile.herokuapp.com/api/graphql'
+# - export CMS_MEDIA='https://joplin-pr-3244-guide-icon-tile.herokuapp.com/media'
+
+# Or,  maybe you want to use staging or production media
+# - Staging Media: `https://joplin-austin-gov-static.s3.amazonaws.com/staging/media`
+# - Production Media: `https://joplin-austin-gov-static.s3.amazonaws.com/production/media`
+
+export NODE_PATH='./src'
+
+export REACT_STATIC_PREFETCH_RATE=5
+# Must be 127.0.0.1, not localhost
```

Comment:
might be cool to have a note as to why that's the case here

Reply:
Well. I don't actually know why that's the case. But I will make a note that it will break if we use "localhost".